templates/zerver/app/keyboard_shortcuts.html

```diff
@@ -194,7 +194,7 @@
                 </thead>
                 <tr>
                     <td class="definition">{% trans %}Edit your last message{% endtrans %}</td>
-                    <td><span class="hotkey"><kbd>Left</kbd></span></td>
+                    <td><span class="hotkey"><kbd class="arrow-key">⇽</kbd></span></td>
```

Comment:
← has better font support than ⇽, is more consistent with ↑ and ↓, and I think is more readable too. Similarly with → over ⇾.

Reply:
Thanks for catching this @andersk, I did a blind copy from /help and didn't notice. I think this is something we'd like to change everywhere so I made a separate commit which does that, hope that's okay.